wnprc_billing/resources/queries/ehr/aliasesWithInvestigators.sql

```diff
@@ -0,0 +1,12 @@
+SELECT
+    alias,
+    investigatorId,
+    (CASE WHEN investigatorId.lastName IS NOT NULL AND investigatorId.firstName IS NOT NULL
+          THEN (investigatorId.lastName ||', '|| investigatorId.firstName)
+          WHEN investigatorId.lastName IS NOT NULL AND investigatorId.firstName IS NULL
+          THEN investigatorId.lastName
+          WHEN investigatorId.lastName IS NULL AND investigatorId.firstName IS NULL
```

Comment:
Make this an ELSE instead, in case firstName has a value but last name is null

Reply:
Please review latest commit related to this.